src/Runner.Worker/Handlers/ContainerActionHandler.cs

```diff
@@ -130,7 +130,7 @@ public async Task RunAsync(ActionRunStage stage)
                 {
                     if (!string.IsNullOrEmpty(arg))
                     {
-                        container.ContainerEntryPointArgs = container.ContainerEntryPointArgs + $" \"{arg.Replace("\"", "\\\"")}\"";
+                        container.ContainerEntryPointArgs = container.ContainerEntryPointArgs + $" \"{arg.Replace("\\", "\\\\").Replace("\"", "\\\"")}\"";
```

Comment:
Can we update this pr to make this same change to all the `ScriptHandlers` and lock it behind the feature flag
`DistributedTask.EnhancedRunnerEscaping`